Dockerfile

```
@@ -23,6 +23,10 @@ COPY . .
 # Build the application
 RUN npm install
 
+# Install dependencies for nunjucks linting
+RUN apt-get update && apt-get install -y \
+  pip && pip install djlint
```

```suggestion
  pip && pip install djlint==1.31.1
```

Non-blocking suggestion: Might be good to explictly list the version of `djlint` is required, in case a future change breaks it.

---

This is a good suggestion, agreed!

---

Added